javascript/javascript3/week2/movie homework folder/index.js

```diff
@@ -0,0 +1,41 @@
+const badMoviesCatalogue = document.getElementById("badMovies");
+const badMoviesPost2000Catalogue = document.getElementById("badMoviesYears");
+
+const url = "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+
+const movies = fetch(url)
+.then((response)=>response.json())
+.then((movies)=>{
+    const badMoviesTitle = movies 
+.filter((movie)=>movie.rating <4)
+.map((movie)=>`<li>${movie.title}</li>`);
+badMoviesCatalogue.innerHTML = `Bad movies catalogue: ${badMoviesTitle.join("")}`;
+
+const badMoviesPost2000Title = movies 
+.filter((movie)=>movie.rating <4 && movie.year>=2000)
+.map((movie)=>`<li>${movie.title}</li>`);
+badMoviesPost2000Catalogue.innerHTML = `Bad movies post2000 catalogue: 
+${badMoviesPost2000Title.join("")}`;
+
+})
+
+const badMoviesTally = async()=>{
+    const response = await fetch (url);
+    const movies = await response.json();
+    const badMovies = movies.filter((movie)=>movie.rating<4);
+    const badMoviesPost2000 = movies.filter((movie)=>movie.year>=2000);
```

Comment:
you should make the second filter on the badMovies array so you got both bad and since 2000

Reply:
and you dont need another function to log out the result for instance if you add console.log(badMovies)
after  line 25  and console.log(badMoviesPost2000)after line 26 ,,then you just need to call badMoviesTally()
think to writ less